crates/esthri-cli/tests/integration/http_server.rs

```diff
@@ -292,3 +307,78 @@ fn test_fetch_archive_with_compressed_files() {
         key_hash_pairs,
     );
 }
+
+#[tokio::test]
```

Comment:
Please add comment describing what the test aims to validate, including whether this addresses nominal behavior or edge cases if applicable.